python/scripts/delete_lambdas.py

```diff
@@ -0,0 +1,11 @@
+"""Delete all Lambda functions in a region.
```

Comment:
This script seems dangerous. I think we should have a warning and question in the beginning.

Reply:
I changed it so that it asks about each function before deleting (does that work?).

Reply:
d05ae11